src/sql/workbench/contrib/query/browser/gridPanel.ts

```diff
@@ -558,7 +560,6 @@ export abstract class GridTableBase<T> extends Disposable implements IView {
 		this.selectionModel.onSelectedRangesChanged.subscribe(e => {
 			this.actionBar.context = this.generateContext();
 		});
-		this.rebuildActionBar();
```

Comment:
if you remove this, the query editor won't have actionbar displayed, right?

Reply:
Moved the action bar changes its own PR: https://github.com/microsoft/azuredatastudio/pull/16178